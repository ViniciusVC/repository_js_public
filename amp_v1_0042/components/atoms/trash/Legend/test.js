import React from "react";
import { shallow } from "enzyme";
import Component from "./index";

describe("Legend >", () => {
  it("deve renderizar com o tema default", () => {
    const wrapper = shallow(<Component />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com a cor primaria", () => {
    const wrapper = shallow(<Component primary={true} />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com a cor gradiente", () => {
    const wrapper = shallow(<Component gradient={true} />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
});
