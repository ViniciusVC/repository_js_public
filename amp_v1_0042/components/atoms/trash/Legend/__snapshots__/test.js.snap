// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legend > deve renderizar com a cor gradiente 1`] = `
<legend
  className="sc-bdVaJa dHluzZ"
/>
`;

exports[`Legend > deve renderizar com a cor primaria 1`] = `
<legend
  className="sc-bdVaJa bILcnh"
/>
`;

exports[`Legend > deve renderizar com o tema default 1`] = `
<legend
  className="sc-bdVaJa jelemA"
/>
`;
